js/hoot/control/utilities/Folder.js

```diff
@@ -139,6 +139,32 @@ Hoot.control.utilities.folder = function(context) {
             });
         }
 
+        function updateLastAccessed(node) {
+            var row = d3.select(node.parentNode);
+            row.classed('expiring', true)
+                .append('g').append('svg:foreignObject')
+                .attr('class', 'expiring')
+                .attr('transform', function(d) {
+                    var dd = d.depth-1;
+                    var dy=772+(11*dd);
+                    return 'translate('+ dy +',-9)'; 
```

Comment:
The timer positioning gets janky if we resize the window.
![image](https://user-images.githubusercontent.com/2600932/33576639-81676360-d90e-11e7-8f29-ee54c8aa13f0.png)